src/Payum/Paypal/ExpressCheckout/Nvp/Api.php

```diff
@@ -303,10 +312,11 @@ class Api
     );
 
     /**
-     * @param array                $options
+     * @param array                    $options
      * @param HttpClientInterface|null $client
+     * @param MessageFactory|null      $messageFactory
      */
-    public function __construct(array $options, HttpClientInterface $client = null)
+    public function __construct(array $options, HttpClientInterface $client = null, MessageFactory $messageFactory = null)
```

Comment:
Should this still be optional? If so, we need to require discovery in composer.json.